templates/web/base/around/_report_banner.html

```diff
@@ -2,5 +2,5 @@ <h1 class="big-green-banner">
     [% loc( 'Click map to report a problem' ) %]
 </h1>
 <p class="map-alternatives">
-    [% tprintf( loc('Can’t use the map? <a href="%s" rel="nofollow">Skip this step.</a>'), url_skip) %]
+    [% tprintf( loc('Can’t use the map to start a report? <a href="%s" rel="nofollow">Skip this step →</a>'), url_skip) %]
```

Comment:
Should this be encoded as `&rarr;`? Genuine question, I don't know the answer :) 

Also, does this have implications for RTL support? i.e. in a right-to-left localisation should this point left?

Reply:
Could we possibly keep the same text in both situations to keep one translation? Unless we think it makes a big difference, would `Can't use the map to start a report? <a href="%s" rel="nofollow">Skip this step</a>` (no arrow, no full stop) be okay in both places? That deals with the RTL issue (which I think is, yes, the arrow in the green banner swaps round automatically, for example).

Reply:
I think that should work as well, it won't have much difference. I added a fixup for it.
Let me know if that is ok